src/shape/geometric_shapes_utility.cpp

```diff
@@ -94,10 +94,7 @@ std::vector<Vec3f> getBoundVertices(const Ellipsoid& ellipsoid, const Transform3
 
   std::vector<Vec3f> result(12);
 
-  const FCL_REAL phi = (1.0 + sqrt(5.0)) / 2.0;  // golden ratio
-  // TODO: Replace with constexpr when we migrate to C++11 or
-  //       boost::math::constants::phi<FCL_REAL>() if boost version is greater
-  //       than 1.50.
+  const FCL_REAL phi = (1.0 + std::sqrt(5.0)) / 2.0;  // golden ratio
```

Comment:
I found that none of the math functions in C++11 are not allowed to use with `constexpr` (gcc allows it, but [it seems not to be allowed in C++14](http://stackoverflow.com/a/27744080)). Okay, we can't use `constexpr` here.